### PR TITLE
Update description for COLLECTION_OBJECT_RESPAWN_SPEED_RATE

### PR DESCRIPTION
**Pull request by gchai**
Updated descriptions for collectable respawn time.
It's not obvious through the documentation that a higher time means a longer respawn period.

### CODE REVIEW

**Review by thejcpalma — APPROVED** [review]

**Comment by jammsen** [comment]
LGTM - After finishing merging #160 / #179 i can merge this with develop.

**Comment by jammsen** [comment]
Hey @gchai - Can you sync again please?

**Pull request closed without merging**
